### DISCUSSION TITLE
FYI/for your lunch break: visualisation of the development of cwa-server

### DISCUSSION THREAD

**Discussion by vaubaehn** [corona-warn-app/cwa-server#1460] [Show and tell]
Dear server devs,

I found this nice animation on twitter, and I hope it is ok to share this with you in this context, and connected with a huge THANK YOU for all your enagagement in the past year. ❤️ 

Happy CWA anniversary! 🎂 

v.

https://twitter.com/i/status/1402354382704893952

**Comment by heinezen** [comment]
@vaubaehn 

Made with https://gource.io/ if you ever want to try this yourself 😉 

----
Corona-Warn-App Open Source Team
